doc/hardware/porting/shields.rst

```diff
@@ -8,6 +8,293 @@ to extend its features and services for easier and modularized prototyping.
 In Zephyr, the shield feature provides Zephyr-formatted shield
 descriptions for easier compatibility with applications.
 
+Shield activation
+*****************
+
+Activate support for one or more shields by adding the matching ``--shield`` arguments
+to the west command:
+
+  .. zephyr-app-commands::
+     :app: your_app
+     :board: your_board
+     :shield: x_nucleo_idb05a1,x_nucleo_iks01a1
+     :goals: build
+
+
+Alternatively, it could be set by default in a project's CMakeLists.txt:
+
+.. code-block:: cmake
+
+	set(SHIELD x_nucleo_iks01a1)
+
+
+Shield interfaces
+*****************
+
+A shield is defined by two key characteristics:
+
+#. ``Physical connectors`` - the mechanical interface
+#. ``Electrical signals`` - what each pin actually does
```

Comment:
```suggestion
#. **Physical connectors** - the mechanical interface
#. **Electrical signals** - what each pin actually does
```


Not sure what I was thinking, sorry :/
I'll approve anyway and we can fix as a follow-up, it really isn't a big deal